.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,18 @@
+<!--
+- Please fill in this template according to the PR you're about to submit.
+- Features and deprecations must be submitted against the master branch.
+- Bug fixes must be submitted against the lowest branch where they apply
+  (lowest branches are regularly merged to upper ones so they get the fixes too).
+-->
+
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
+| Branch?       | master / 2.7, 2.8 or 3.1 <!--see comment above-->
 | Bug fix?      | yes/no
 | New feature?  | yes/no
 | BC breaks?    | yes/no
 | Deprecations? | yes/no
 | Tests pass?   | yes/no
-| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
+| Fixed tickets | #...
 | License       | MIT
-| Doc PR        | reference to the documentation PR, if any
+| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
```

Comment:
Nope, because this is a link to the `symfony/symfony-docs`, not an actual reference to a ticket in this repo